qmlbind/src/api_application.cpp

```diff
@@ -91,6 +91,11 @@ qmlbind_application qmlbind_application_new(int argc, char **argv)
     return app;
 }
 
+qmlbind_application qmlbind_application_instance(void)
+{
+    return static_cast <QApplication*> (QCoreApplication::instance());
```

Comment:
Yeah, I can make that change, the documentation I was looking at only mentioned the `instance()` static method.